types/config.ts

```diff
@@ -163,6 +167,12 @@ export const ConnectElementCustomMethodConfig = {
     setOnApplicationSubmitted: (_listener: (() => void) | undefined): void => {}
   },
   "capital-financing-promotion": {
-    setOnApplicationSubmitted: (_listener: (() => void) | undefined): void => {}
+    setLayout: (_layout: "full" | "banner" | undefined): void => {},
```

Comment:
I recommend defining the `FinancingLayoutType` similar to how you defined `FinancingProductType`, so it can be shared with the react repo

Reply:
@jorgea-stripe updated! (Although I called it `FinancingPromotionLayoutType` - happy to update to `FinancingLayoutType` if you have strong opinions)